src/include/btmem.h

```diff
@@ -1340,6 +1340,8 @@ struct __wt_ref {
     WT_REF_HIST hist[WT_REF_SAVE_STATE_MAX];
     uint64_t histoff;
 #endif
+
+    wt_shared volatile uint16_t ref_changes;
```

Comment:
Can you also add a comment to explain this field?

Reply:
Yes, I've now added a comment.